content/links.md

```diff
@@ -408,3 +471,52 @@ instance_groups:
       - port
       - url
 ```
+___
+
+## Possible Link Scenarios
+
+The rules for uniqueness are described in the tables below:
+
+Providers can be specified in any way you want as long as it is **not** consumed.
+
+| Provider Name | Provider Type | Consumer Type | Allowed |
+|---------------|---------------|---------------|---------|
+| Same          | Same          | No consumer   | True    |
```

Comment:
Is `Same` an example or mean the same as something else? Similar question about `Different` below?

Reply:
Will update and provide examples.